### ISSUE TITLE
Is the manual included in the compiled package for Windows?

### ISSUE THREAD

**Issue by GlassGruber** [qpdf/qpdf#521]
Hello, I've downloaded the windows package for the latest release but I can't find the compiled manual in the `doc/` folder, only a `stylesheet.css` file. Not really a big issue since it's easily available online, but was wondering if this is intended.

Thank you for this incredibly useful software!

**Comment by jberkenbilt** [comment]
You're right. It's missing. I'll make sure this is fixed for the next release. Thanks for bringing it to my attention!
